adapter/src/main/java/org/jvalue/ods/adapterservice/adapter/api/rest/v1/Mappings.java

```diff
@@ -1,10 +1,8 @@
 package org.jvalue.ods.adapterservice.adapter.api.rest.v1;
 
 public class Mappings {
-  public static final String DATA_PATH = "/data";
-  public static final String PREVIEW_PATH = "/preview";
   public static final String IMPORT_PATH = "/dataImport";
-  public static final String RAW_PREVIEW_PATH = "/preview/raw";
+  public static final String RAW_IMPORT_PATH = "/dataImport/raw";
```

Comment:
No reason to distinguish between preview and import anymore since everything is stateless now

Reply:
You're right. I think the name preview is more precisely though to indicate no downstream processing.

Reply:
I agree, preview fits better. We should think about changing the method type to GET as well. However, I think this should be done in another pr

Reply:
I think changing to GET won't do since we are sending a body.

Reply:
Good point, I did not think about that